tests/rptest/clients/kafka_cli_tools.py

```diff
@@ -431,3 +435,22 @@ def _execute(self, cmd):
     def _script(self, script):
         version = self._version or KafkaCliTools.VERSIONS[0]
         return "/opt/kafka-{}/bin/{}".format(version, script)
+
+    def oauth_produce(self, topic: str, num_records: int, acks=-1):
+        path = "/opt/strimzi-kafka-oauth/examples/producer"
+        assert self._oauth_cfg is not None, "Must provide an OAuthConfig at construction"
+        cmd = [
+            "java",
+            "-Dsecurity.protocol=sasl_plaintext",
+            f"-Dacks={acks}",
+            f"-Dbootstrap.servers={self._redpanda.brokers()}",
+            f"-Doauth.token.endpoint.uri={self._oauth_cfg.token_endpoint}",
+            f"-Doauth.client.id={self._oauth_cfg.client_id}",
+            f"-Doauth.client.secret={self._oauth_cfg.client_secret}",
+            "-cp",
+            f"{path}/target/*:{path}/target/lib/*",
+            "io.strimzi.examples.producer.ExampleProducer",
+            str(topic),
```

Comment:
nit: according to the param type hints, the topic is already a string so there's no need to convert.

Reply:
Oh, unless this is in-case someone passes `TopicSpec` then this makes sense.

Reply:
Nope, this is copy/paste cruft